src/test/java/com/sun/javatest/StatusTest.java

```diff
@@ -0,0 +1,25 @@
+package com.sun.javatest;
```

Comment:
Missing license. Probably copy it from the [APITest](https://github.com/jtulach/netbeans-apitest/blob/master/src/test/java/org/netbeans/apitest/APITest.java) file.